app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -29,8 +29,8 @@ def course_choice_rows(application_choice)
         course_row(application_choice),
         study_mode_row(application_choice),
         location_row(application_choice),
-        type_row(application_choice.course),
-        course_length_row(application_choice.course),
+        type_row(application_choice),
+        course_length_row(application_choice),
```

Comment:
Not introduced in this PR, but this component is really confusing. In some places it uses the offered_choice and in some other places it's referencing the course_choice e.g. `location_row`'s action is mapped to `{application_choice.course.name_and_code`. I wonder if this file needs some refactoring so it's easier to understand what is actually going on here.

Reply:
👍 changed to use `offered_course` throughout